client/relay-chain-minimal-node/src/network.rs

```diff
@@ -113,246 +107,56 @@ pub(crate) fn build_collator_network(
-		&mut self,
-		_who: PeerId,
-		_response: sc_network_common::sync::message::BlockResponse<B>,
-	) -> Result<sc_network_common::sync::OnBlockJustification<B>, sc_network_common::sync::BadPeer>
-	{
-		unimplemented!("Not supported on the RPC collator")
-	}
-
-	fn on_justification_import(
-		&mut self,
-		_hash: <B as BlockT>::Hash,
-		_number: polkadot_service::NumberFor<B>,
-		_success: bool,
-	) {
-	}
-
-	fn on_block_finalized(
-		&mut self,
-		_hash: &<B as BlockT>::Hash,
-		_number: polkadot_service::NumberFor<B>,
-	) {
-	}
-
-	fn push_block_announce_validation(
-		&mut self,
-		_who: PeerId,
-		_hash: <B as BlockT>::Hash,
-		_announce: sc_network_common::sync::message::BlockAnnounce<<B as BlockT>::Header>,
-		_is_best: bool,
-	) {
-	}
-
-	fn poll_block_announce_validation(
-		&mut self,
-		_cx: &mut std::task::Context,
-	) -> std::task::Poll<sc_network_common::sync::PollBlockAnnounceValidation<<B as BlockT>::Header>>
-	{
-		std::task::Poll::Pending
-	}
-
-	fn peer_disconnected(&mut self, _who: &PeerId) {}
-
-	fn metrics(&self) -> sc_network_common::sync::Metrics {
-		Metrics {
-			queued_blocks: 0,
-			fork_targets: 0,
-			justifications: sc_network_common::sync::metrics::Metrics {
-				pending_requests: 0,
-				active_requests: 0,
-				importing_requests: 0,
-				failed_requests: 0,
-			},
-		}
-	}
-
-	fn block_response_into_blocks(
-		&self,
-		_request: &sc_network_common::sync::message::BlockRequest<B>,
-		_response: sc_network_common::sync::OpaqueBlockResponse,
-	) -> Result<Vec<sc_network_common::sync::message::BlockData<B>>, String> {
-		unimplemented!("Not supported on the RPC collator")
-	}
+struct SyncOracle;
 
-	fn poll(
-		&mut self,
-		_cx: &mut std::task::Context,
-	) -> std::task::Poll<PollBlockAnnounceValidation<B::Header>> {
-		std::task::Poll::Pending
+impl sp_consensus::SyncOracle for SyncOracle {
+	fn is_major_syncing(&self) -> bool {
+		false
 	}
 
-	fn send_block_request(&mut self, _who: PeerId, _request: BlockRequest<B>) {
-		unimplemented!("Not supported on the RPC collator")
+	fn is_offline(&self) -> bool {
+		true
 	}
-
-	fn num_active_peers(&self) -> usize {
-		0
-	}
-
-	fn process_block_response_data(&mut self, _blocks_to_import: Result<OnBlockData<B>, BadPeer>) {}
-}
-
-struct DummyChainSyncService<B>(std::marker::PhantomData<B>);
-
-impl<B: BlockT> NetworkSyncForkRequest<B::Hash, NumberFor<B>> for DummyChainSyncService<B> {
-	fn set_sync_fork_request(&self, _peers: Vec<PeerId>, _hash: B::Hash, _number: NumberFor<B>) {}
 }
 
-impl<B: BlockT> JustificationSyncLink<B> for DummyChainSyncService<B> {
-	fn request_justification(&self, _hash: &B::Hash, _number: NumberFor<B>) {}
-
-	fn clear_justification_requests(&self) {}
-}
-
-impl<B: BlockT> Link<B> for DummyChainSyncService<B> {
-	fn blocks_processed(
-		&mut self,
-		_imported: usize,
-		_count: usize,
-		_results: Vec<(Result<BlockImportStatus<NumberFor<B>>, BlockImportError>, B::Hash)>,
-	) {
-	}
+pub fn get_block_announce_proto_config<B: BlockT>(
```

Comment:
Doesn't need to be public or? 
```suggestion
fn get_block_announce_proto_config<B: BlockT>(
```